server/src/controllers/auth.js

```diff
@@ -1,29 +1,32 @@
+/* eslint-disable no-underscore-dangle */
 import axios from 'axios';
+import jwt from 'jsonwebtoken';
 import User, { validateUser } from '../models/user';
 import Config from '../config/config';
 
+export const generateJWT = (userId) => {
+  const { JWT_SECRET } = Config.server;
+  const token = jwt.sign({ id: userId }, JWT_SECRET); // TODO: add expiry time.
```

Comment:
Please add expiry time to some time, we can update later. As such we don't have to change the interface later, only the value will be different.